nncf/tensorflow/api/compression.py

```diff
@@ -67,3 +73,19 @@ def apply_to(self, model: TModel) -> TModel:
             self.initialize(transformed_model)
 
         return transformed_model
+
+    def _get_model_converter_and_graph(self, model: TModel) -> Tuple[TFModelConverter, NNCFGraph]:
```

Comment:
Honestly, I'm going to remove `api` module from the `tensorflow` module. Could you implement `check_scopes_in_graph` without extenshion functionality of the `compression.py`.

Reply:
removed